kitsune/wiki/models.py

```diff
@@ -1101,7 +1101,7 @@ class ImportantDate(ModelBase):
 # the M2M tables, etc.
 class Locale(ModelBase):
     """A localization team."""
-    locale = models.CharField(max_length=7, db_index=True)
+    locale = LocaleField(db_index=True)
```

Comment:
The bits in the DB won't change, but I think South still wants a migration for stuff like this. It should make one with `./manage.py schemamigration wiki --auto`.